examples/browser-js/lib-proto/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ protobuf {
     }
     plugins {
         id("kotlin") {
```

Comment:
Update this from `kotlin` to `pbandk` in all of the examples, as well as the `"kotlin"` reference in the `task.plugins` section below

Reply:
I fixed by commit 89a73e9